### DISCUSSION TITLE
ScriptReference/Time-unscaledDeltaTime

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#210] [Notes]
# ScriptReference/Time-unscaledDeltaTime

https://docs.unity3d.com/ScriptReference/Time-unscaledDeltaTime.html

**Comment by mvi** [comment]
Note that unscaledDeltaTime does not always behave the same as deltaTime even when timeScale is 1. unscaledDeltaTime will increase when the game is in the background, if the game isn't updating for 10 seconds then when you come back unscaledDeltaTime will be about 10 seconds whereas deltaTime will likely be a very small value. [More details](https://stackoverflow.com/questions/46503824/time-unscaleddeltatime-still-counting-even-when-app-is-in-background).
